### ISSUE TITLE
[Proposal] Permission-based middleware

### ISSUE THREAD

**Issue by gcpatton** [laraflock/dashboard#6]
A permission-based middleware with wildcards would be great!

IE: Permissions with the slug of `post.create` and `post.update` would be covered by

``` php
<?php
Route::get('/posts', array(
    'uses' => 'PostController@index',
    'middleware' => 'permission:post.*'
));
```

Also, passing an array to the middleware parameters would be nice as well

`'middleware' => 'permission:post.create,post.update,dashboard.*'`

**Comment by iolson** [comment]
I will definitely look into adding this, as I think its a great update and needed.

**Comment by iolson** [comment]
@gcpatton I have updated a PermissionMiddleware class on master branch. Once I finish some other updates I will merge into 3.0 and release.

**Comment by iolson** [comment]
Fixed in v3.1.0
